lib/master.js

```diff
@@ -84,9 +87,14 @@ class Master extends EventEmitter {
 
     this.ready(() => {
       this.isStarted = true;
-      const stickyMsg = this.options.sticky ? ' with STICKY MODE!' : '';
-      this.logger.info('[master] %s started on %s (%sms)%s',
-        frameworkPkg.name, this[APP_ADDRESS], Date.now() - startTime, stickyMsg);
+      if (this.options.nginxSticky) {
+        this.logger.info('[master] %s started on %j (%sms) with NGINX STICKY MODE!',
+          frameworkPkg.name, this[WORKER_ADDRESSES].map(getAddress), Date.now() - startTime);
+      } else {
+        const stickyMsg = this.options.sticky ? ' with STICKY MODE!' : '';
+        this.logger.info('[master] %s started on %s (%sms)%s',
+          frameworkPkg.name, this[APP_ADDRESS], Date.now() - startTime, stickyMsg);
+      }
 
```

Comment:
What's the different between `nginxSticky` and `sticky`? Seems they are mixed together.

Reply:
- `nginxSticky`: nginx -> app-worker
- `sticky`: nginx -> master -> app-worker